### ISSUE TITLE
Windows wheels fail to build due to ssize_t redefinition

### ISSUE THREAD

**Issue by chayim** [redis/hiredis-py#150]
```
        C:\Users\runneradmin\AppData\Local\pypa\cibuildwheel\Cache\nuget-cpython\pythonx86.3.7.9\tools\include\pyconfig.h(167): note: see declaration of 'ssize_t'
        D:\a\hiredis-py\hiredis-py\vendor\hiredis\sds.h(38): error C2371: 'ssize_t': redefinition; different basic types
        C:\Users\runneradmin\AppData\Local\pypa\cibuildwheel\Cache\nuget-cpython\pythonx86.3.7.9\tools\include\pyconfig.h(167): note: see declaration of 'ssize_t'
        D:\a\hiredis-py\hiredis-py\vendor\hiredis\sds.h(105): warning C4244: 'return': conversion from 'uint64_t' to 'size_t', possible loss of data
        D:\a\hiredis-py\hiredis-py\vendor\hiredis\sds.h(130): warning C4244: 'return': conversion from 'uint64_t' to 'size_t', possible loss of data
        D:\a\hiredis-py\hiredis-py\vendor\hiredis\sds.h(198): warning C4244: 'return': conversion from 'uint64_t' to 'size_t', possible loss of data
        src\pack.c(91): warning C4244: '=': conversion from '__int64' to 'Py_ssize_t', possible loss of data
        error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\VC\\Tools\\MSVC\\14.29.30133\\bin\\HostX86\\x86\\cl.exe' failed with exit code 2
        [end of output]
```

[Complete output](https://github.com/redis/hiredis-py/actions/runs/4042448459/jobs/6950538448#step:4:174)

**Comment by chayim** [comment]
cc @prokazov @zalmane

**Comment by chayim** [comment]
closed by #151
